cli/Cargo.toml

```diff
@@ -52,13 +52,13 @@ tokio = { version = "^1.0", features = ["full"] }
 toml = "0.5.8"
 unicode-width = "0.1.9"
 url = { version = "2", features = ["serde"] }
-zip = "0.6.2"
+zip = { version = "0.6.2", default-features = false, features = [] }
```

Comment:
The latest version of zip doesn't actually build without disabling its password support, since the rust crypto folks have a habit of yanking old versions and zip seems rather poorly maintained.

The master is up to date, but unfortunately no release has been made since then. But it seems like we don't need all those extra features anyway, since we just want to extract zips.